homeassistant/components/roborock/quality_scale.yaml

```diff
@@ -64,7 +64,7 @@ rules:
     status: exempt
     comment: There are no noisy entities.
   entity-translations: done
-  exception-translations: todo
+  exception-translations: done
```

Comment:
It feels like maybe we shouldn't mark this done until `action-exceptions` is done too